Content.Server/_NF/Smuggling/DeadDropSystem.cs

```diff
@@ -116,15 +121,36 @@ private void SendDeadDrop(EntityUid uid, DeadDropComponent component, EntityUid
         dropHint.AppendLine(dropLocation.ToString());
         dropHint.AppendLine();
         dropHint.AppendLine(Loc.GetString("deaddrop-hint-posttext"));
+        var printout = new FaxPrintout(dropHint.ToString(),Loc.GetString("deaddrop-hint-name"),null,null);
+        var isFax = HasComp<FaxMachineComponent>(uid);
+        if (isFax == true)
+        {
+            _faxSystem.Receive(uid,printout,null,null);
+            _adminLogger.Add(LogType.Action, LogImpact.Medium, $"sent a dead drop to {dropLocation.ToString()} from {ToPrettyString(uid)} at {Transform(uid).Coordinates.ToString()}");
+        }
+        else
+        {
+            var paper = EntityManager.SpawnEntity(component.HintPaper, Transform(uid).Coordinates);
 
-        var paper = EntityManager.SpawnEntity(component.HintPaper, Transform(uid).Coordinates);
-
-        _paper.SetContent(paper, dropHint.ToString());
-        _meta.SetEntityName(paper, Loc.GetString("deaddrop-hint-name"));
-        _meta.SetEntityDescription(paper, Loc.GetString("deaddrop-hint-desc"));
-        _hands.PickupOrDrop(user, paper, handsComp: hands);
+            _paper.SetContent(paper, dropHint.ToString());
+            _meta.SetEntityName(paper, Loc.GetString("deaddrop-hint-name"));
+            _meta.SetEntityDescription(paper, Loc.GetString("deaddrop-hint-desc"));
+            _hands.PickupOrDrop(user, paper, handsComp: hands);
+            _adminLogger.Add(LogType.Action, LogImpact.Medium, $"{ToPrettyString(user)} sent a dead drop to {dropLocation.ToString()} from {ToPrettyString(uid)} at {Transform(uid).Coordinates.ToString()}");
 
+        }
         //reset the timer
         component.NextDrop = _timing.CurTime + TimeSpan.FromSeconds(_random.Next(component.MinimumCoolDown, component.MaximumCoolDown));
+        return true;
     }
+
+    private void OnDeadDropPrinterUsed(EntityUid uid, DeadDropComponent comp, ref GotDropPrintedEvent args)
+    {   TryComp<HandsComponent>(args.UserUid, out var hands);
+        if (hands == null || HasComp<FaxMachineComponent>(args.TargetUid) == false)
+        {
+            return;
+        }
+        args.Handled = SendDeadDrop(args.TargetUid, comp, uid, hands);
```

Comment:
Shouldn't _really_ need a hand component in this case.  Maybe either the argument should be nullable and you pass null, or you have a fax-specific version of SendDeadDrop that omits it.